lib/addons/external-secrets/index.ts

```diff
@@ -40,6 +40,12 @@ const defaultIamPolicy: iam.PolicyStatement = new iam.PolicyStatement({
     "secretsmanager:DescribeSecret",
     "secretsmanager:ListSecretVersionIds",
     "secretsmanager:ListSecrets",
+    "ssm:DescribeParameters",
+    "ssm:GetParameter",
+    "ssm:GetParameters",
+    "ssm:GetParametersByPath",
+    "ssm:GetParameterHistory",
+    "kms:Decrypt"
```

Comment:
At some point we will need to review all such policies for wildcard resource and provide an approach to lock it down to specific resources only, like we did in the EBS csi driver.